L1Trigger/VertexFinder/plugins/VertexNTupler.cc

```diff
@@ -429,6 +443,8 @@ namespace l1tVertexFinder {
       }
     }
 
+    //const Vertex& TruePrimaryVertex = inputData.getPrimaryVertex();
```

Comment:
Is this commented out code needed? If so, please add some comment that explains why it is needed. Otherwise just remove